src/metadataparsers/mods/CdmToMods.php

```diff
@@ -113,26 +85,25 @@ private function createCONTENTdmFieldValuesArray($objectInfo)
     }
 
     /**
-     *  Create MODS XML
-     *  @param array $colletionMappyingArray collection mappings
-     *  @param array $objectInfo array of info. about the object that the MODS XML will be created for
+     *  {@inheritdoc}
      */
     public function createModsXML($collectionMappingArray, $objectInfo)
     {
         $CONTENTdmFieldValuesArray = $this->CONTENTdmFieldValuesArray;
 
         $pointer = $objectInfo['pointer'];
 
-        $modsString = '';
-
-        $modsOpeningTag = '<mods xmlns="http://www.loc.gov/mods/v3" ';
-        $modsOpeningTag .= 'xmlns:mods="http://www.loc.gov/mods/v3" ';
-        $modsOpeningTag .= 'xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ';
-        $modsOpeningTag .= 'xmlns:xlink="http://www.w3.org/1999/xlink">';
+        $modsOpeningTag = sprintf(
+            '<mods xmlns="%s" xmlns:mods="%s" xmlns:xsi="%s" xmlns:xlink="%s">',
+            MODS::$MODS_NAMESPACE_URI,
+            MODS::$MODS_NAMESPACE_URI,
+            "http://www.w3.org/2001/XMLSchema-instance",
+            "http://www.w3.org/1999/xlink"
+        );
```

Comment:
One thing I love about Fedora is they define all the namespaces in the code, so you can just reference them instead of having to ensure it is typed correctly each time.

You could create a Utility class with a bunch of statically named namespaces for re-use in your codebase.

I draw your attention to this example from static-ldp 
https://github.com/trellis-ldp/static-ldp/blob/master/src/TrellisConstants.php